src/ray/gcs/gcs_server/gcs_init_data.cc

```diff
@@ -27,6 +27,8 @@ void GcsInitData::AsyncLoad(const EmptyCallback &on_done) {
     }
   };
 
+  AsyncLoadWorkerTableData(on_load_finished);
```

Comment:
Sorry, I don't quite understand your question. Could you please provide some additional information

Reply:
We use counter at L21 (just several lines above) for synchronization.
Since you add an extra operation, you should increment the counter value from 5 to 6.

Reply:
Comments have been added, thx.